bin/base

```diff
@@ -751,3 +751,27 @@ function init_es() {
 
     return ${RC}
 }
+
+function confirm_continue() {
+    local ref
+    local action_message
+    local selection
+
+    typeset -n ref=${1}
```

Comment:
-n is bad option for me, are you sure this works ?

Reply:
it does, assuming you have a new enough bash.  we are actually forcefully installing a newer version of bash into the client-server containers in order to support this syntax already (since our oldest usernevs don't support it natively)

Reply:
Additionally, in this situation a prompt is being printed and then the answer is being captured so we can't use traditional bash output capture since that would include the prompt.